cmd/rekor_monitor/main.go

```diff
@@ -145,10 +138,16 @@ func main() {
 			}
 		}
 
+		err = rekor.RunConsistencyCheck(rekorClient, verifier, *logInfoFile)
```

Comment:
For a follow up - I noticed `RunConsistencyCheck` calls `GetLogInfo`, so we could reduce duplication by having `RunConsistencyCheck` return the log info struct.

Reply:
will work on this in a follow-up PR now